docs/app/partials/layout-introduction.tmpl.html

```diff
@@ -0,0 +1,364 @@
+<div ng-controller="LayoutCtrl" class="layout-content" ng-cloak>
+
+  <h3>Overview</h3>
+  <p>
+    Angular Material's Layout features are purposed as syntactic HTML/CSS sugar to allow developers
```

Comment:
"Angular Material's Layout features are purposed as syntactic HTML/CSS sugar to allow developers..."
>
"Angular Material's Layout features provide sugar to enable developers to more easily create modern, responsive layouts on top of CSS3 flexbox. The layout API consists of a set of Angular directives that can be applied to any of your application's HTML content."